### DISCUSSION TITLE
regarding CA files windows to linux

### DISCUSSION THREAD

**Discussion by gfshsfgsfgfsgh** [Chia-Network/chia-blockchain#6143] [Support]
so I run GUI on windows and am trying to transfer over to gui on debian. By transfer over I mean just plug the hard drives on the windows machine into the Linux one. I've already got the blockchain and everything updated on the linux gui.

Bash doesn't see chia as a command and I don't know where to cd to. Maybe I'm just flat out doing it wrong for linux. I tried doing the equivalent of `cd C:\Users(your username)\AppData\Local\Chia-Blockchain\app-1.1.1\resources\app.asar.unpacked\daemon` (ignore version) for linux. So I can do the `chia init -c` thing.

Do I need to do it at all? Normally I would just trial and error but I want to have as little downtime as possible.

thanks

**Comment by tiny0little** [comment]
how did you install on linux? what linux flavor you use?

**Reply by gfshsfgsfgfsgh** [reply]
the .deb
debian

---edit---

yeah, I installed properly and now cli works fine. Makes sense in hindsight. Installing via the .deb only installs the gui functionality.

**Comment by BrandtH22** [comment]
@gfshsfgsfgfsgh , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the detailed instructions for farming on many machines can be found here https://docs.chia.net/farming-on-many-machines

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
